src/sidebar/test/annotation-fixtures.js

```diff
@@ -54,6 +56,7 @@ function newAnnotation() {
     references: [],
     text: 'Annotation text',
     tags: ['tag_1', 'tag_2'],
+    user: 'acct:bill@localhost',
```

Comment:
Please note this change!

The "meaning" of a "new annotation" per the annotation's _components_ has shifted some as a result of the changes in this PR. 

Before: Other annotations kind of "relied on" the `annotation` controller to inject some defaults into an annotation object, and this is exacerbated by imperfect unit tests/modeling in some of our legacy tests (we're getting much better!). It was also problematic because current `annotation` controller tests (there are lots, as it's a big component) assume the presence of some fields _because the controller itself_ previously injected them, whereas "now" the annotation controller retrieves an annotation object with defaults injected by the store.

The addition of the `user` property here is the bare minimum alteration to these fixtures that allow current tests to pass. It could be a useful exercise to vet fixtures at some point to make sure they're really what we think they are.

For example, these fixtures don't make explicit what "context" they apply to. Is it a "new annotation" per the perspective of what gets thrown over to the wall for us by the `annotator`? Per retrieving an annotation from the service that doesn't correspond to any annotations the store knows about yet? Per the `annotation` controller getting a "new" (unsaved) annotation after expected defaults have been injected?

Didn't want to break off these deeper thoughts in this PR, but they did occur to me...